typescript/cli/src/commands/registry.ts

```diff
@@ -96,6 +112,34 @@ const addressesCommand: CommandModuleWithContext<{ name: string }> = {
   },
 };
 
+const rpcCommand: CommandModuleWithContext<{
```

Comment:
Can you try writing a e2e test for it? Just copy the boilerplate from warp-apply.e2e-test.ts

Reply:
Or https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/e115bc1df91370085c1cb16a409d166524f6cada/typescript/cli/src/tests/warp-read.e2e-test.ts

Reply:
given this doesnt really interact with the chain I personally feel like an e2e test is a bit overkill, wdyt?